spec/getCountriesSpec.js

```diff
@@ -135,4 +138,120 @@ describe("The index", () => {
       ).toBeTrue();
     });
   });
-});
+  
+  describe("The getCountriesByAlcoholProhibition where parameter is none", () => {

```

Comment:
I think that different argument for same _API_ Method is the same _test_ but different _test case_. We should have a single test (`describe()`) with multiple test cases (`it()`) for specific _API_ Method.

You can have a look at the other tests to _"match the pattern"_ 😉

Reply:
Sure thing! will do that.